tests/test_base_client.py

```diff
@@ -314,3 +315,140 @@ def mock_iter_content(chunk_size):
     assert received_data == test_data # all data was received correctly
     assert len(content_chunks) == expected_chunks # correct number of chunks
     assert all(len(c) <= chunk_size for c in content_chunks) # chunks don't exceed size
+
+
+def test_is_seekable_stream():
+    client = _BaseClient()
+
+    # Test various input types that are not streams.
+    assert not client._is_seekable_stream(None) # None
+    assert not client._is_seekable_stream("string data") # str
```

Comment:
Should we internally wrap static strings and byte arrays via `BytesIO` to make them seekable and so easily uploadable?

Reply:
Done.